sentences/pl/homeassistant_HassGetState.yaml

```diff
@@ -18,6 +18,14 @@ intents:
             - scene
             - script
 
+      - sentences:
+          - stan <name> [<area>]
+        response: one_state
+        excludes_context:
+          domain:
+            - scene
+            - script
+
       - sentences:
           - jak(i|a|e) jest status <name> [<area>]
```

Comment:
Could this please cover both `status` and `stan` at the same time?

Reply:
don't know if i did this right, I just grouped sentences